mne/time_frequency/tfr.py

```diff
@@ -1107,9 +1162,245 @@ def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
+                    title = 'Aggregate (%s) of %s' % (aggregate, ch_title)
+
             if title:
                 fig.suptitle(title)
 
+            plt_show(show)
+            return fig, data, times*1e-3, freqs
+
+    @verbose
+    def plot_joint(self, picks=None, baseline=None, mode='mean',
+             tmin=None, tmax=None, fmin=None, fmax=None, vmin=None, vmax=None,
+             cmap='RdBu_r', dB=False, colorbar=True, show=True, title=None,
+             layout=None, yscale='auto', aggregate='mean', exclude=None,
+             timefreqs=None, timefreqs_average=None, topomap_args=None,
+             verbose=None):
+        """Plot TFRs as a two-dimensional image(s) with topomaps
+
+        Parameters
+        ----------
+        picks : None | array-like of int
+            The indices of the channels to plot, one figure per channel. If
+            None, plot the across-channel aggregate.
+        baseline : None (default) or tuple of length 2
+            The time interval to apply baseline correction.
+            If None do not apply it. If baseline is (a, b)
+            the interval is between "a (s)" and "b (s)".
+            If a is None the beginning of the data is used
+            and if b is None then b is set to the end of the interval.
+            If baseline is equal ot (None, None) all the time
+            interval is used.
+        mode : None | 'ratio' | 'zscore' | 'mean' | 'percent' | 'logratio' | 'zlogratio'
+            Do baseline correction with ratio (power is divided by mean
+            power during baseline) or zscore (power is divided by standard
+            deviation of power during baseline after subtracting the mean,
+            power = [power - mean(power_baseline)] / std(power_baseline)),
+            mean simply subtracts the mean power, percent is the same as
+            applying ratio then mean, logratio is the same as mean but then
+            rendered in log-scale, zlogratio is the same as zscore but data
+            is rendered in log-scale first.
+            If None no baseline correction is applied.
+        tmin : None | float
+            The first time instant to display. If None the first time point
+            available is used.
+        tmax : None | float
+            The last time instant to display. If None the last time point
+            available is used.
+        fmin : None | float
+            The first frequency to display. If None the first frequency
+            available is used.
+        fmax : None | float
+            The last frequency to display. If None the last frequency
+            available is used.
+        vmin : float | None
+            The mininum value of the color scale. If vmin is None, the data
+            minimum value is used.
+        vmax : float | None
+            The maxinum value of the color scale. If vmax is None, the data
+            maximum value is used.
+        cmap : matplotlib colormap | 'interactive' | (colormap, bool)
```

Comment:
Good point! No, it doesn't work yet. Maybe for the moment I'll just remove it from the docstring, what do you think?